### DISCUSSION TITLE
Is it possible to disable the new fail screen?

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#15226] [Q&A]
I updated lazer today and now there's a very loud (visually) red flashy popup on fail (the one added in #15122, I think). this is quite scary and not great in general. is there any way it could be disabled? I can't find a setting for it.

**Comment by bdach** [comment]
You will be able to disable the red flash specifically using the "Fade playfield to red when health is low" setting next release after https://github.com/ppy/osu/pull/15175. But outright disabling isn't really an option, I don't think.

**Reply by ghost** [reply]
good enough for me, thanks!

**Reply by ghost** [reply]
hi :wave: I updated lazer today and disabled that option. it's a little bit better, but the animation and audible sound are still really not good, is there anything that could be done about this?

**Reply by peppy** [reply]
What's your specific issue? If it's "scary" maybe you should change your sound effect volume. Eventually this will be skinnable (it might already be?)

**Reply by ghost** [reply]
it reminds me a lot of a gunshot, with the sudden moving the whole screen back. I tried turning the sound effect volume down, and it really doesn't help.
I'll take a look at the skin settings
